Source/WebKit/UIProcess/API/APIDataTask.cpp

```diff
@@ -47,8 +47,10 @@ void DataTask::setClient(Ref<DataTaskClient>&& client)
 
 void DataTask::cancel()
 {
-    if (m_networkProcess && m_sessionID && m_identifier)
-        m_networkProcess->cancelDataTask(*m_identifier, *m_sessionID);
+    if (RefPtr networkProcess = m_networkProcess.get()) {
```

Comment:
I think we should do the opposite to avoid an unnecessary ref/deref if m_session or m_identifier are null opt:
```
if (m_sessionID && m_identifier) {
    if (RefPtr networkProcess = m_networkProcess.get())
        networkProcess->cancelDataTask(*m_identifier, *m_sessionID);
}
```